src/main/java/rocks/nt/apm/jmeter/config/influxdb/InfluxDBConfig.java

```diff
@@ -113,6 +128,13 @@ public InfluxDBConfig(BackendListenerContext context) {
 			influxRetentionPolicy = DEFAULT_RETENTION_POLICY;
 		}
 		setInfluxRetentionPolicy(influxRetentionPolicy);
+		
+		String influxHTTPScheme = context.getParameter(KEY_HTTP_SCHEME, DEFAULT_HTTP_SCHEME);
+		if (StringUtils.isEmpty(influxHTTPScheme)) {
+			influxRetentionPolicy = DEFAULT_HTTP_SCHEME;
```

Comment:
this assignment doesn't look correct:
influxRetentionPolicy = DEFAULT_HTTP_SCHEME

Reply:
Sorry typo. Fixed!